<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripPlanner</class>
 <widget class="QDialog" name="TripPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TripPlanner</string>
  </property>
  <widget class="QGroupBox" name="info_gb_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Trip Information</string>
   </property>
   <widget class="QLabel" name="time_label_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Approximate Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="from_label_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="to_label_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="arrival_rb_">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrival</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="departure_rb_">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Departure</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_edit_">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_edit_">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="from_cb_">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New York</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Berlin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beijing</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="to_cb_">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New York</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Berlin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beijing</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QProgressBar" name="status_pb_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>23</height>
    </size>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="result_tw_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Departure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arrival</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Duration</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="status_label_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
